### ISSUE TITLE
Crash with api 'com.braintreepayments.api:drop-in:6.0.1'  and  'com.braintreepayments.api:data-collector:4.10.0'

### ISSUE THREAD

**Issue by bm185178** [braintree/braintree-android-drop-in#338]
### General information

* SDK/Library version: 6.0.1
* Environment: Preprod and production
* Android Version and Device: all

### Issue description

When drop-in sdk version 6 will be updated to support 4.10.0? Right now it is crashing with
```
 java.lang.NoSuchMethodError: No virtual method getAuthorization()Lcom/braintreepayments/api/Authorization; in class Lcom/braintreepayments/api/BraintreeClient; or its super classes (declaration of 'com.braintreepayments.api.BraintreeClient' appears 
```

**Comment by stwonary** [comment]
We are in the process to migrate to Android SDK v4 and are stuck on this issue too.

**Comment by sarahkoop** [comment]
Hello - we are in the process of updating Drop-in for the latest version of the Braintree core SDK. We have a PR in progress with the changes (#337). We will post an update when these changes have been released. Thanks for your patience!

**Comment by bm185178** [comment]
Any update?

**Comment by huync21** [comment]
it's still crashing

**Comment by sshropshire** [comment]
@bm185178 @huync21 Thank you for your patience. We're working on getting the release out soon, but there are some outstanding items we need to address first before moving forward.

We'll report back here once the update has been released.

**Comment by mattwylder** [comment]
This has been released in 6.1.0
